CommonCompilerConfig.cmake

```diff
@@ -36,35 +36,37 @@
 #                               source files; e.g., for cmake `add_definitions()`
 # COMMON_COMPILER_DEFINITIONS_STR: string variant of COMMON_COMPILER_DEFINITIONS
 #
-# COMMON_C_FLAGS_UNOPTIMIZED: string of common C flags without explicit optimization flags.
+# EXTERNAL_C_FLAGS_UNOPTIMIZED: string of C flags without explicit optimization flags for 3rd-party sources
```

Comment:
I agree that it would be cleaner to keep the comments under 80 wherever possible, despite cmake files often going beyond 80 in other areas where it's impractical to keep them under. I will revisit it next time I'm in this neck of the woods.